extensions/simon/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "simon",
+  "title": "Simon",
+  "description": "Play a game of Simon.",
+  "icon": "command-icon.png",
+  "author": "andreaselia",
+  "categories": [
+    "Fun"
+  ],
+  "license": "MIT",
+  "commands": [{
+    "name": "index",
+    "title": "Play Simon",
+    "subtitle": "Simon",
```

Comment:
```suggestion
```
Redundant since it's already a part of the title

Reply:
Removed